ecommers/lib/core/provider_models/products_grid_provder_model.dart

```diff
@@ -72,7 +68,7 @@ class ProductsGridProviderModel extends ProviderModelBase
   }
 
   Future<List<Product>> _fetchMoreProducts(int from, int to) {
-    return services.productService.fetchProducts(
```

Comment:
the reason why I added services alias to the package is that paginator field from the dependency_services conflicts with the paginator from ItemsLoadingNotifier:)

Reply:
Seems now everything is ok, am I right?
Also maybe better to rename it accordingly?